src/install/python/python-agent-license.mdx

```diff
@@ -0,0 +1,10 @@
+---
+componentType: default
+headingText: Get your license key
+---
+
+You'll need a valid New Relic license key to install the Python agent. License keys are available to those who have a New Relic account, so if you don't already have a New Relic account, [sign up](https://newrelic.com/signup?via=login). It's free, forever!
```

Comment:
I'd recommend adding the new license key component here.

Reply:
@paperclypse where is the documentation for that?